course-mw/course-actors-common/src/main/java/org/sunbird/learner/actors/course/CourseManagementActor.java

```diff
@@ -23,10 +23,12 @@
 
 import static org.sunbird.common.models.util.JsonKey.EKSTEP_BASE_URL;
 import static org.sunbird.common.models.util.ProjectUtil.getConfigValue;
-import static org.sunbird.common.responsecode.ResponseCode.SERVER_ERROR;
+import static org.sunbird.common.responsecode.ResponseCode.CLIENT_ERROR;
+import static org.sunbird.common.models.util.JsonKey.CONTENT_PROPS_TO_ADD;
 
 public class CourseManagementActor extends BaseActor {
     private static ObjectMapper mapper = new ObjectMapper();
+    private static List<String> metadataToBeAdded = Arrays.stream(getConfigValue(CONTENT_PROPS_TO_ADD).split(",")).map(String::trim).collect(Collectors.toList());
```

Comment:
Add empty list incase config property is not available. Also check for empty list while using **metadataToBeAdded** for filtration. If this list is empty throw client exception.

Reply:
Also add one unit test case for nullity and empty check.